cv/pothole_detection/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(pothole_detection)
+
+# # Find catkin macros and libraries
+find_package(catkin REQUIRED COMPONENTS 
+  rospy 
+  std_msgs 
+  geometry_msgs
+  )
+
+# Generate cmake config files via catkin_package
+catkin_package(
+  INCLUDE_DIRS
+  CATKIN_DEPENDS
+  )
+  
```

Comment:
Fix end of line (look up how to do it in whaever text editor you are using)

Reply:
this is from catkin pkg create? not sure what it's wanting here